core/models.py

```diff
@@ -388,33 +419,33 @@ class Hackathon(MetaDataMixin):
         help_text="Location of the hackathon, I for in-person, V for virtual, H for hybrid",
     )
 
-    # Boolean fields for whether the hackathon is specific to certain groups
-    is_web3 = models.BooleanField(default=False)  # Web 3 hackathons
-    is_diversity = models.BooleanField(
-        default=False
-    )  # Diversity hackathons for specific marginalized groups
-    is_restricted = models.BooleanField(
-        default=False
-    )  # Hackathons with restricrted enrollment (ex. only students of some university)
-    is_nonenglish = models.BooleanField(
-        default=False
-    )  # Hackathons which are not in English
-    is_over18 = models.BooleanField(
-        default=False
-    )  # Hackathons which are only for people over 18
-
     freeze_data = models.BooleanField(
         default=False,
         help_text="Set to True to not update any details using scraped data. Use if you get accurate details directly from the hackathon organizers.",
     )
 
+    is_web3 = models.BooleanField(
+        default=False, help_text="Is the hackathon Web3 themed"
+    )
+    is_diversity = models.BooleanField(
+        default=False, help_text="Is the hackathon only for underrepresented groups"
+    )
+    is_restricted = models.BooleanField(
+        default=False,
+        help_text="Is enrollment in this hackathon restricted to only some group of people (like those enrolled in one specific school or unoversity)",
+    )
+    is_nonenglish = models.BooleanField(
+        default=False, help_text="Is the primary language of this hackathon not English"
+    )
+
     custom_info = models.JSONField(
         default=dict, null=True, blank=True
     )  # Anything else that we might want to add in a structured format
 
+    vote_count = models.BigIntegerField(default=0)
```

Comment:
hmm... I would rename to rating or something like net_vote as it's not really a count but a sum of the negative and positive votes